src/algebra/module/submodule_lattice.lean

```diff
@@ -157,6 +158,33 @@ lemma mem_Sup_of_mem {S : set (submodule R M)} {s : submodule R M}
   (hs : s ∈ S) : ∀ {x : M}, x ∈ s → x ∈ Sup S :=
 show s ≤ Sup S, from le_Sup hs
 
+/-- This is the statement that `coe : (submodule R M) → set M` is Scott continuous for the
+ω-complete partial order induced by the complete lattice structures. See
+`omega_complete_partial_order.continuous`. -/
+@[norm_cast, simp] lemma coe_supr_of_chain_eq_supr_coe (a : ℕ →ₘ submodule R M) :
+  (↑(⨆ k, a k) : set M) = ⋃ k, (a k : set M) :=
```

Comment:
Isn't this statement just a weaker version of `submodule.coe_supr_of_directed`? It seems like we're missing a `preorder_hom.directed` lemma to provide the glue.

Reply:
Oh, thanks! I don't know how I didn't find `coe_supr_of_directed`.